File: apps/problems/Calculate-Fibonacci/boilerplate-full/function.cpp

            #include <stdio.h>
            #include <stdlib.h>
            #include <stdbool.h>
            #include <string.h>
            #include <limits.h>
            
            ##USER_CODE_HERE##
            
            int main() {
                FILE *file = fopen("/dev/problems/Calculate-Fibonacci/tests/inputs/##INPUT_FILE_INDEX##.txt", "r");
                vector<string> lines;
                string line;
                while (fgets(line, sizeof(line), file)) lines.push_back(line);
                fclose(file);
                int n;
  fscanf(file, "%d", &n);
                int result = calculateFibonacci(n);
                fprintf(stdout, "%d\n", result);
                return 0;
                }
            

File: apps/problems/Calculate-Fibonacci/boilerplate-full/function.rs
use std::fs::read_to_string;
            use std::io::{self};
            use std::str::Lines;

            ##USER_CODE_HERE##

            fn main() -> io::Result<()> {
            let input = read_to_string("/dev/problems/Calculate-Fibonacci/tests/inputs/##INPUT_FILE_INDEX##.txt")?;
            let mut lines = input.lines();
            let n: i32 = lines.next().unwrap().parse().unwrap();
            let result = calculateFibonacci(n);
            println!("{}", result);
            Ok(())
            }
            